Lib/inspect.py

```diff
@@ -509,12 +510,22 @@ def isbuiltin(object):
         __self__        instance to which a method is bound, or None"""
     return isinstance(object, types.BuiltinFunctionType)
 
+def ismethodwrapper(object):
+    """Return true if the object is a method wrapper.
+
+    For example;
+
+    >>> ismethodwrapper(object().__str__)
+    ... True"""
```

Comment:
Seems like the example is redundant (considering the rest of the similiar methods don't have one)
```suggestion
    """Return true if the object is a method wrapper."""
```